doc/developer/platform/architecture-cloud.md

```diff
@@ -32,10 +32,44 @@ The STORAGE and COMPUTE layers each turn on instances by creating services in
 an [`Orchestrator`]. An `Orchestrator` is backed by Kubernetes in the cloud,
 and a service corresponds directly to a `StatefulSet`. This may evolve soon.
 
+All ports created by the orchestrator have names.
+
+The orchestrator installs the following labels on all services and pods that it
+creates:
+
+  * `materialized.materialize.cloud/namespace`: the orchestrator "subnamespace"
+    in which the service is created. Either `storage` or `compute` at present.
+
+  * `materialized.materialize.cloud/service-id`: the layer-provided identifier
+    for the service, like "cluster-0" or "runtime".
```

Comment:
Is this something we can match on for Linkerd policies, or something more specific within the layer?

Should we be matching on `materialized.materialize.cloud/namespace`?

Reply:
It's a good question. I updated the labeling strategy to give you some flexibility. There's now a `materialized.materialize.cloud/port-NAME` label for every named port, which strikes me as the right thing to match on for now. I also introduced a `compute.materialized.materialize.cloud/type=cluster` label in case you want to match on all compute clusters in the compute subnamespace without relying on the fact that presently _all_ pods in the compute subnamespace are cluster pods.